lib/generateTasks.js

```diff
@@ -48,11 +57,26 @@ module.exports = function generateTasks({
         // match both `test.js` and `subdirectory/test.js`.
         matchBase: !pattern.includes('/')
       }
-    ).map(file => normalize(relative ? file : path.resolve(cwd, file)))
+    )
+      .map(file => (relative ? file : path.resolve(cwd, file)))
+      .map(file => normalize(file))
```

Comment:
This should probably be last for better Windows support.

Reply:
👍 Will Do

Reply:
Hmmm that changes the results of the tests